icu4j/main/tests/core/src/com/ibm/icu/dev/data/units/unitsTest.txt

```diff
@@ -0,0 +1,197 @@
+# This file is a copy of common/testData/units/unitsTest.txt from CLDR.
+# WIP/TODO(hugovdm): determine a good update procedure and document it.
```

Comment:
We have a location now for automatically updated tests, please use this:
icu4j/main/tests/core/src/com/ibm/icu/dev/data/cldr/units/unitsTest.txt